server/main.py

```diff
@@ -165,6 +175,8 @@ async def index(request):
                 'script': ' '.join(script),
                 'description': params['description'],
                 'output': output_dir,
+                'hail': hail_version,
```

Comment:
Just to make this less ambiguous:

```suggestion
                'hailVersion': hail_version,
```